### DISCUSSION TITLE
Generic Huggingface GPT-2 model

### DISCUSSION THREAD

**Discussion by cahya-wirawan** [facebookresearch/ParlAI#4357] [Ideas]
Hi,
I am trying to finetune Huggingface gpt2 model for my own language. As I understand from the source code of hugggingface agent, the model that can be used is limited only to few standard english gpt-2 models (small, medium, large and xl). There is no possibility to use other gpt2 models. Therefore I add this functionality https://github.com/cahya-wirawan/ParlAI/commit/6b081c2e6faff22029cf55ad5eb065832ffbd43a , where I add a new argument (--model_name) to set any existing Huggingface gpt-2 models. If this functionality is not done yet before (I am new in ParlAI, so maybe I overlook it) and if it could be helpful for other, I will create a PR.

**Comment by klshuster** [comment]
Yes, please submit a PR!
